.github/workflows/publish.yml

```
@@ -22,7 +22,7 @@ jobs:
           python-version: "3.11"
 
       - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==2.12.3
+        run: python -m pip install cibuildwheel==2.17.0
```

```suggestion
        run: python -m pip install cibuildwheel==2.18.0
```
Was just released on Saturday, might as well use it